vhd/packer/windows-2019-vars.json

```diff
@@ -2,6 +2,6 @@
     "image_publisher": "MicrosoftWindowsServer",
     "image_offer": "WindowsServer",
     "image_sku": "2019-Datacenter-Core-smalldisk",
-    "image_version": "17763.1518.2010132039",
+    "image_version": "17763.1697.2101090203",
```

Comment:
since the image is already available in Azure we could use the image and not have to install the patches:

```
az vm image show --urn MicrosoftWindowsServer:WindowsServer:2019-Datacenter-Core-smalldisk:latest --query name
"17763.1757.2102060435"
```